changelog/unreleased/2623.md--compression-level-config.md

```diff
@@ -0,0 +1,6 @@
+There is a new setting, `vast.zstd-compression-level`, to control the
+compression level of the zstd algorithm used in both the feather and parquet
+store backend. The default value is `1`, which already was the default for the
+feather store backend, but is a change for parquet, which previously had its
+own default of `9`. We find `1` to be a very good trade-off between performance
+and storage size.
```

Comment:
I'm not sure we should mention that the default is 1, because that isn't even what we set it to in the code. The default that we use is the default that Apache Arrow has (or recommends, in a way). It could change in future Arrow versions.

Reply:
That's a valid point. However, if we chose to do so then we are shipping a software with a software knob we don't know the default value for and / or it depends on the specific arrow version, which is not exactly user friendly.

Reply:
There are quite a few default values for which we do not know a concrete value. It's just "the default," or "automatically chosen." This applies to quite a few CAF-related tuning options, for example. I think we should be accurate here, and saying "the default level for Apache Arrow" is in my opinion.